File: CSIT111/Lecture_2/Lecture_Labs/Lecturer_Examples/TestOperators.java
// The use of / and % operators
// 
// File name: TestOperators.java

class TestOperators
{
	public static void main (String [ ] args)
	{
		int m = 1234;
		
		// How to get the last digit of m?
		int lastDigit = m % 10;
		System.out.println ("The last digit of " + m + " is " + lastDigit);
		
		// What about the last 2 digits of m?
		int last2Digits = m % 100;
		System.out.println ("The last 2 digits of " + m + " are " + last2Digits);	
		
		// How to remove the last digit of m?
		System.out.print ("After removing the last digit of " + m);
		m = m / 10;			// Integer division
		System.out.println (", the new integer is " + m);
	}
}

File: CSIT111/Lecture_2/Lecture_Labs/Lecturer_Examples/TestOperators.py
class TestOperators:
    def main(self):
        
        m = 1234
        
        # How to get the last digit of m?
        lastDigit = m % 10
        print("The last digit of ",m," is ",lastDigit)
        
        # What about the last 2 digits of m?
        last2Digits = m % 100
        print("The last 2 digits of ",m," is ",last2Digits)
        
        # How to remove the last digit of m?
        print("After removing the last digit of ", m)
        m = m // 10			# Integer division
        print(", the new inter is ", m)
        
testoperators = TestOperators()
testoperators.main()